web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/var/isFunction.js

```diff
@@ -0,0 +1,13 @@
+define( function() {
+	"use strict";
+
+	return function isFunction( obj ) {
+
+      // Support: Chrome <=57, Firefox <=52
```

Comment:
:warning: **Warning**: Found precision alignment of 2 spaces (*WordPress.WhiteSpace.PrecisionAlignment.Found*).